examples/ibm-cloudant/standard-plan-on-dedicated-hw/variables.tf

```diff
@@ -0,0 +1,18 @@
+// Resource arguments for ibmcloud_api_key
+variable "ibmcloud_api_key" {
+  description = "IBM Cloud API key."
+  type        = string
+}
+
+// Resource arguments for service_region
+variable "service_region" {
+  description = "Region in which service has to be provisioned."
+  type        = string
+  default     = "us-south"
+}
+
+// Resource arguments for cloudant_cluster_information
+variable "cloudant_cluster_information" {
+  description = "The actual physical location of the Dedicated Hardware plan instance."
```

Comment:
As a var this is a little removed from the fact that the value is an environment CRN. We could use a better description here, but I wonder if it makes sense to reference the CRN via a terraform data source that is pointing to the cluster resource instance instead of a var value.

e.g. the [docs](https://cloud.ibm.com/docs/Cloudant?topic=Cloudant-creating-and-leveraging-an-ibm-cloudant-dedicated-hardware-plan-instance-on-ibm-cloud#provisioning-a-standard-plan-instance-on-a-dedicated-hardware-environment-with-the-ibm-cloud-cli) say:
>Or you can determine what the CRN is by using the ibmcloud resource service-instance SERVICE_INSTANCE_NAME command.

so I feel like we could similarly extract the CRN from the cluster name - which itself could be defined in the var; which would have a description like "Cloudant dedicated hardware plan service instance name"

I know this is a little tricky because it is virtually impossible for us to test.